src/requests.jl

```diff
@@ -55,21 +55,16 @@ For failed HTTP or etcd requests an `HTTPError` or `EtcdError` is thrown respect
 - `String`: all non-json response bodies
 """
 function request(f::Function, uri::String, opts::Dict; n=5, max_delay=10.0)
-    logger = get_logger(current_module())
+    logger = getlogger(current_module())
 
-    retry_cond(resp) = in(statuscode(resp), 300:400) && haskey(resp.headers, "Location")
+    retry_cond(resp, ex) = in(statuscode(resp), 300:400) && haskey(resp.headers, "Location")
```

Comment:
What is the `ex` for? Does Requests.jl (e.g., `Requests.get`) return something else with the http response now?

Reply:
Looks like the retry condition function now receives the exception that triggered the retry as well.

Reply:
Ahh, okay. I guess I haven't read the docstring for `retry` in a while :)